plugins/manual_classification/routes.py

```diff
@@ -154,18 +158,26 @@ def post(self, arguments):
         db_task.save(commit=True)
 
         # add classification step
-        step_id = "classification"
+        step_id = "annotate-class-1"
         href = url_for(
-            f"{MANUAL_CLASSIFICATION_BLP.name}.ClassificationView", db_id=db_task.id
+            f"{MANUAL_CLASSIFICATION_BLP.name}.ClassificationView",
+            db_id=db_task.id,
+            _external=True,
         )
         ui_href = url_for(
             f"{MANUAL_CLASSIFICATION_BLP.name}.MicroFrontendClassification",
             db_id=db_task.id,
+            _external=True,
         )
+        db_task.data["step_id"] = 1
 
         # all tasks need to know about db id to load the db entry
         task: chain = pre_render_classification.s(db_id=db_task.id) | add_step.s(
-            db_id=db_task.id, step_id=step_id, href=href, ui_href=ui_href, prog_value=30
+            db_id=db_task.id,
+            step_id=step_id,
+            href=href,
+            ui_href=ui_href,
+            prog_value=db_task.data["step_id"] / (db_task.data["step_id"] + 1) * 100,
```

Comment:
please use `()` it makes it more readable and the intent more clear `(a/b)*100` vs. `a/b*100`